README.md

```diff
@@ -15,6 +15,7 @@
 
 8. [Issues and Inquiries](#Issues-and-Inquiries)
 
+9. [Test Cases](#Test-Cases)

```

Comment:
In the main body, "Test Cases" is before "Examples", we better keep the same sequence in the table of contents.